cl/search/migrations/0025_docket_unique_docket_per_court.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN;
+--
+-- Create constraint unique_docket_per_court on model docket
+--
+CREATE UNIQUE INDEX CONCURRENTLY "unique_docket_per_court" ON "search_docket" ((MD5("docket_number")), "pacer_case_id", "court_id");
```

Comment:
According to [our wiki](https://github.com/freelawproject/courtlistener/wiki/Database-migrations#adding-and-removing-indexes-to-a-column-dont-use-concurrently-by-default), I don't think `CONCURRENTLY` can be used in a transaction, can it?

Reply:
You're right! Sorry. I'm fixing this.

Reply:
A small thing!